docs/ref/contrib/gis/geos.txt

```diff
@@ -511,6 +511,16 @@ Topological Methods
     Returns a :class:`GEOSGeometry` representing the points shared by this
     geometry and other.
 
+.. method:: GEOSGeometry.make_valid()
+
+    .. versionadded::4.1
+
+    Returns a valid :class:`GEOSGeometry` equivalent, trying not to lose any of
+    the input vertices. If the geometry is already valid, it is returned
+    untouched. This is similar to the
+    :class:`~django.contrib.gis.db.models.functions.MakeValid` database
+    function.
```

Comment:
Shouldn't this be documented in the `Other Properties & Methods` section? as `normalize()` :thinking: